packages/engine-server/src/doctor/service.ts

```diff
@@ -593,6 +597,59 @@ export class DoctorService implements Disposable {
         };
         break;
       }
+      case DoctorActionsEnum.FIX_INVALID_FILENAMES: {
+        const { canRename, cantRename, stats } = this.findInvalidFileNames({
+          notes,
+          noteDicts: {
+            notesById,
+            notesByFname,
+          },
+        });
+        resp = stats;
+
+        if (canRename.length > 0 && !opts.quiet) {
+          console.log("Found invalid filename in notes:\n");
```

Comment:
ideally, we wouldn't be using `console.log` here but the [print](https://github.com/dendronhq/dendron/blob/master/packages/dendron-cli/src/commands/base.ts#L255:L255) function from the `CLICommand`. can we pass it in as an optional parameter in the constructor? by default, its a no-op.

Reply:
also apply to other `console.log` statements